### PR TITLE
Particular point topology is sober; and totally disconnected implies sober

### PR DESCRIPTION
**Pull request by Almanzoris**

### CODE REVIEW

File: theorems/T000608.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+uid: T000608
+if:
+  P000047: true
+then:
+  P000073: true
+---
+
+Suppose that $C$ is an irreducible non-empty closed subset of $X$. As $X$ is {P47}, then $C$ is a singleton. Therefore, $X$ is sober.
```
Comment (StevenClontz) [inline theorems/T000608.md:9]:
```suggestion
Suppose that $C$ is a {P39} non-empty closed subset of $X$. As $X$ is {P47}, then $C$ is a singleton. Therefore, $X$ is {P73}.
```

And similar for other two files

**Comment by prabau** [comment]
T608.  Nice result.  Allows to derive that two more spaces are sober.

Minor grammar point: No "then" should be used after an "As ..." clause.

**Comment by Almanzoris** [comment]
Thanks!

> T608. Nice result. Allows to derive that two more spaces are sober.
> 
> Minor grammar point: No "then" should be used after an "As ..." clause.

Going to fix that.

File: spaces/S000008/properties/P000192.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+space: S000008
+property: P000192
+value: true
+---
+
+$X$ is closed, {P39} and $p$ is a generic point in $X$. Let $C$ be a proper non-empty closed subset of $X$, $C$ is a subset of $X \setminus \\{p\\}$, which is a discrete subspace, hence {P47}. Therefore, $C$ is hyperconnected if and only if $C$ is a singleton. It follows that $X$ is {P192}.
```
Comment (StevenClontz) [inline spaces/S000008/properties/P000192.md:7]:
```suggestion
$X$ is closed, {P39} and $p$ is a generic point in $X$. Let $C$ be a proper non-empty closed subset of $X$, $C$ is a subset of $X \setminus \{p\}$, which is a discrete subspace, hence {P47}. Therefore, $C$ is {P39} if and only if $C$ is a singleton. It follows that $X$ is {P192}.
```

File: spaces/S000009/properties/P000192.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+space: S000009
+property: P000192
+value: true
+---
+
+$X$ is closed, {P39} and $p$ is a generic point in $X$. Let $C$ be a proper non-empty closed subset of $X$, $C$ is a subset of $X \setminus \\{p\\}$, which is a discrete subspace, hence {P47}. Therefore, $C$ is hyperconnected if and only if $C$ is a singleton. It follows that $X$ is {P192}.
```
Comment (StevenClontz) [inline spaces/S000009/properties/P000192.md:7]:
```suggestion
$X$ is closed, {P39} and $p$ is a generic point in $X$. Let $C$ be a proper non-empty closed subset of $X$, $C$ is a subset of $X \setminus \{p\}$, which is a discrete subspace, hence {P47}. Therefore, $C$ is {P39} if and only if $C$ is a singleton. It follows that $X$ is {P192}.
```

**Review by StevenClontz — APPROVED** [review]

**Comment by StevenClontz** [comment]
Note that https://topology.pi-base.org/spaces?q=%7ELocally+Hausdorff%2BTotally+disconnected is open and might strengthen T608.

**Comment by StevenClontz** [comment]
I think https://topology.pi-base.org/spaces/S000186 is totally disconnected. It is constructed to fail locally Hausdorff.

**Comment by Almanzoris** [comment]
@StevenClontz

In the beginning of the proof, I assumed that $X$ isn't empty, but the empty space is totally disconnected according to $\pi$-Base. Maybe I should have said something like: "Suppose that $X$ isn't empty and $C$ is a {P39} non-empty closed subset of $X$."?

**Comment by StevenClontz** [comment]
No need to suppose $X$ isn't empty; if it's empty, your proof is still vacuously valid.

**Comment by Almanzoris** [comment]
Alright. I had to make sure.

**Pull request merged**